com.tobagin.whakarere.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.tobagin.whakarere
```

Comment:
`tobagin.com` doesn't resolve. This is not valid.

Reply:
changed to com.mudeprolinux.whakarere as I own mudeprolinux.com although it doesn´t currently have a hosted website.